.github/workflows/autobuild.yml

```diff
@@ -410,36 +413,16 @@ jobs:
         id:                         staple-macOS-app
         uses:                       devbotsxyz/xcode-staple@ae68b22ca35d15864b7f7923e1a166533b2944bf
         with:
-          product-path:             deploy/${{ steps.get-artifacts.outputs.artifact_1 }}
+          product-path:             publish/${{ steps.get-artifacts.outputs.artifact_1 }}
 
-      - name:                       Upload Artifact 1 to Release
-        # Every build job has at least one artifact. Therefore, no `if artifact_1` condition here.
-        # If the artifact is missing, this should fail.
+      - name:                       Deploy Artifacts to Release
         if:                         needs.create_release.outputs.publish_to_release == 'true'
-        id:                         upload-release-asset1
-        uses:                       actions/upload-release-asset@v1
-        env:
-          GITHUB_TOKEN:             ${{ secrets.GITHUB_TOKEN }}
-        with:
-          upload_url:               ${{ needs.create_release.outputs.upload_url }}
-          asset_path:               deploy/${{ steps.get-artifacts.outputs.artifact_1 }}
-          asset_name:               ${{ steps.get-artifacts.outputs.artifact_1 }}
-          asset_content_type:       application/octet-stream
-
-      - name:                       Upload Artifact 2 to Release
-        if:                         >-
-                                    steps.get-artifacts.outputs.artifact_2 != '' &&
-                                    needs.create_release.outputs.publish_to_release == 'true'
-        id:                         upload-release-asset2
-        uses:                       actions/upload-release-asset@v1
+        id:                         upload-release-assets
+        run:                        cd publish; gh release upload "${{ needs.create_release.outputs.tag_name }}" ${{ steps.get-artifacts.outputs.artifact_1 }} ${{ steps.get-artifacts.outputs.artifact_2 }} --clobber
```

Comment:
Not sure if omitting quotes is the best way to "overload" this command. Basically if quotes are omitted we can have an arbitrary amount of files to upload